scripts/qiita-auto-processing

```diff
@@ -161,7 +179,9 @@ for pipeline in full_pipelines:
                      if c['previous-step'] == commands[0]['command-name']]
 
     # Step 3. Find all preparations/artifacts that we can add the pipeline
-    artifacts_all = [a for study in Study.iter()
+    #         ... as a first pass we will only process study 10317 (AGP) ...
+    # artifacts_all = [a for study in Study.iter()
+    artifacts_all = [a for study in [Study(10317)]
```

Comment:
Is this OK to use for deploy in the main system?

Reply:
yup, the idea is that in the main deploy we will _only_ process AGP automatically; once we test multiple times we can add to full qiita.

Reply:
Thanks!